Testscripts/Linux/xfstesting.sh

```diff
@@ -169,6 +169,15 @@ ConfigureCIFS() {
 }
 
 Main() {
+    # Get VM kernel version. Recommend to 3.12 or later for SMB3.0 in Azure.
+    mj=$(uname -r | cut -d '-'  -f1 | cut -d '.' -f 1)
+    mn=$(uname -r | cut -d '-'  -f1 | cut -d '.' -f 2)
+    if [ $mj -lt 4 ] && [ $mn -lt 12 ]; then
```

Comment:
Here we cannot use kernel version directly. Distro providers (like RHEL) backports the patches to old kernel.  Add first level check for distro and then kernel.

As per the document it is supported from RH7.5+, so the base kernel version would be ```3.10.0-862```

Reply:
If distro is RHEL or CentOS, then we set the baseline to 3.10.0. OK?

Reply:
What is "the document"? Where do you find this kernel version? 3.10.0-862

Reply:
https://access.redhat.com/articles/3078

Reply:
This has RHEL 7.5 errata date with this kernel version. But I am still confused about your comment.
Why do you think we should support RHEL 7.5 and this kernel? I reviewed the contents in 2018-04-10 RHEA-2018:0700 but could not find any clue for SMB3.0 bug fix/change.
I still need your clarification why you mentioned "As per the document it is supported from RH7.5+". @johnsongeorge-w 
I can not find patch/bug fix information yet.

Microsoft document is not clearly defined the distro version. "7.5+" is ambiguious. Also, SMB 3.0 encryption is added in 4.11

Reply:
We looked into the database result, and CentOS/RHEL .7.6 or later version passed this test. From the regression perspective, I put the filter for rhel/centos 7.6+.